components/kyma-environment-broker/internal/broker/plans.go

```diff
@@ -42,7 +42,7 @@ var plans = map[string]struct {
 			  "items": [
 				{
 				  "type": "string",
-				  "enum": ["Kiali", "Jaeger"]
+				  "enum": ["Kiali", "Jaeger", "KnativeProvisionerNatss", "NatssStreaming"]
```

Comment:
please revert as described above in main.go file. Because when you are adding that under plans definiton then user can enable those compnent when sending provision request just by adding in body `"components": ["KnativeProvisionerNatss", "NatssStreaming"]` which will mean that both components should be installed

Reply:
Reverted